lua/nvim-treesitter/incremental_selection.lua

```diff
@@ -100,7 +100,7 @@ function M.attach(bufnr)
     if funcname == "init_selection" then
       mode = 'n'
     else
-      mode = 'v'
+      mode = 'x'
```

Comment:
You missed changing the corresponding detach. But everything else LGTM! :+1:  Thanks!

Reply:
sorry, fixed it.